### DISCUSSION TITLE
[@types/ui-grid] update ui-grid to version 4.11.1

### DISCUSSION THREAD

**Discussion by prx-lmo** [DefinitelyTyped/DefinitelyTyped#59213] [Issues with a @types package] [answered]
https://github.com/prx-lmo/DefinitelyTyped/tree/master/types/ui-grid
This version added following new functions:
getRowsByKey(isInEntity, key, comparator, lookInRows)
findRowByKey(isInEntity, key, comparator, lookInRows)
selectRowByKey(isInEntity, key, comparator, evt, lookInRows)
unSelectRowByKey(isInEntity, key, comparator, evt, lookInRows)

[a comment by an automated account is not shown]

**Comment by prx-lmo** [comment, marked as answer]
Fixed here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59214

**Reply by Grayson-code** [reply]
Well it did fail ci tests

**Comment by Grayson-code** [comment]
tho why did u create this thread?
